docs/Tutorial_Advanced_Annotation_Vectors.ipynb

```diff
@@ -0,0 +1,981 @@
+{
```

Comment:
<p>I can see that the EOG data is different between the two tutorials but it may not be obvious to the reader. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/408/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>Oh this is <strong>ECG</strong> data though, not the <strong>EOG</strong> data used in the first tutorial.</p>

Reply:
Ha! Never mind then